convert2rhel/data/7/x86_64/configs/oracle-7-x86_64.cfg

```diff
@@ -46,6 +46,10 @@ default_rhsm_repoids = rhel-7-server-rpms
 # not used. Delimited by any whitespace(s).
 eus_rhsm_repoids =
 
+# List of Extended Lifecycle Support (ELS) repoids to enable through subscription-manager when the --enablerepo option is
+# not used. Delimited by any whitespace(s).
+els_rhsm_repoids =
```

Comment:
We don't need to define the repoids for oracle?

Reply:
Yes I wasn't sure of the repoid ID yet so I set this PR to Draft - will add the proper repoid for all el7 systems